docs/en/getting-started/install.md

```diff
@@ -28,21 +28,23 @@ The quickest and easiest way to get up and running with ClickHouse is to create
 For production installs of a specific release version see the [installation options](#available-installation-options) down below.
 :::
 
-On Linux and macOS:
+On Linux, macOS and FreeBSD:
 
-1. If you are just getting started and want to see what ClickHouse can do, the simplest way to download ClickHouse locally is to run the following command. It downloads a single binary for your operating system that can be used to run the ClickHouse server, clickhouse-client, clickhouse-local,
-ClickHouse Keeper, and other tools:
+1. If you are just getting started and want to see what ClickHouse can do, the simplest way to download ClickHouse locally is to run the
+   following command. It downloads a single binary for your operating system that can be used to run the ClickHouse server,
+   clickhouse-client, clickhouse-local, ClickHouse Keeper, and other tools:
 
   ```bash
   curl https://clickhouse.com/ | sh
   ```
 
 1. Run the following command to start the ClickHouse server:
+
     ```bash
     ./clickhouse server
     ```
 
-    The first time you run this script, the necessary files and folders are created in the current directory, then the server starts.
+   The first time you run this script, the necessary files and folders are created in the current directory, then the server starts.
 
 1. Open a new terminal and use the **clickhouse-client** to connect to your service:
```

Comment:
```suggestion
1. Open a new terminal and use the **./clickhouse client** to connect to your service:
```

Reply:
unless they are on a system where install works they need to use `./clickhouse client`

Reply:
Oh, didn't know that (and I should know :-) )